### PR TITLE
Add clarity to docs surrounding command line arguments, suggest data-dir when syncing, and polishing up for mainnet

### PR DESCRIPTION
**Pull request by TennisBowling**
it's all docs. I added a mainnet reference on the systemd page, added clarity to how command line arguments are given, suggested if they want to specify a data-dir on the syncing page (and also reminded them that they need to pass it to every command)

### CODE REVIEW

File: docs/the_nimbus_book/src/beacon-node-systemd.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ Replace:
 
 `<WEB3-URL>` with the WebSocket JSON-RPC URL that you are planning to use.
 
+> Want to run on mainnet? just replace all instances of `prymont` to `mainnet`.
```
Comment (unixpi) [inline docs/the_nimbus_book/src/beacon-node-systemd.md]:
```suggestion
> If you want to run on mainnet, simply replace all instances of `prymont` with `mainnet`.
```

File: docs/the_nimbus_book/src/start-syncing.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ INF 2020-12-01 11:26:31.000+00:00 Slot start                                 top
 INF 2020-12-01 11:26:36.285+00:00 Slot end                                   topics="beacnde" tid=21815 file=nimbus_beacon_node.nim:593 slot=96567 nextSlot=96568 head=b54486c4:96563 headEpoch=3017 finalizedHead=2f5d12e4:96479 finalizedEpoch=3014
 ...
 ```
-
+> If you want to put the database somewhere else, (e.g. an external ssd) pass the `--data-dir=/your/path`. ⚠️ If you are doing this, remember to pass this flag to **all** your nimbus calls.
```
Comment (unixpi) [inline docs/the_nimbus_book/src/start-syncing.md]:
```suggestion
> If you want to put the database somewhere else, (e.g. an external ssd) pass the `--data-dir=/your/path`. ⚠️ If you do this, remember to pass this flag to **all** your nimbus calls.
```

File: docs/the_nimbus_book/src/troubleshooting.md [hunk 3]
```diff
@@ -50,6 +50,11 @@ If you're experiencing a low peer count, you may be behind a firewall. Try resta
 ```
 ./run-pyrmont-beacon-node.sh --nat:extip:35.124.65.104
 ```
+### noCommand does not accept arguments
+
+When starting nimbus and you get,
```
Comment (unixpi) [inline docs/the_nimbus_book/src/troubleshooting.md]:
```suggestion
If, on start,  you see
```

File: docs/the_nimbus_book/src/troubleshooting.md [hunk 4]
```diff
@@ -50,6 +50,11 @@ If you're experiencing a low peer count, you may be behind a firewall. Try resta
 ```
 ./run-pyrmont-beacon-node.sh --nat:extip:35.124.65.104
 ```
+### noCommand does not accept arguments
+
+When starting nimbus and you get, 
+```The command 'noCommand' does not accept arguments```
+Make sure that your command line flags are in the format of `--foo=bar`, `--baz`, or `--foo-bar=qux`.
```
Comment (unixpi) [inline docs/the_nimbus_book/src/troubleshooting.md]:
```suggestion
Double check to see if your command line flags are in the correct format, i.e. `--foo=bar`, `--baz`, or `--foo-bar=qux`.
```

**Review by unixpi — CHANGES_REQUESTED** [review]

File: docs/the_nimbus_book/src/beacon-node-systemd.md [hunk 5]
```diff
@@ -41,6 +41,7 @@ Replace:
 
 `<WEB3-URL>` with the WebSocket JSON-RPC URL that you are planning to use.
 
+> If you want to run on mainnet, simply replace all instances of `prymont` with `mainnet`.
```
Comment (arnetheduck) [inline docs/the_nimbus_book/src/beacon-node-systemd.md]:
```suggestion
> If you want to run on mainnet, simply replace all instances of `pyrmont` with `mainnet`.
```

**Comment by TennisBowling** [comment]
Review last commit and then ready for merge.

File: docs/the_nimbus_book/src/beacon-node-systemd.md [hunk 6]
```diff
@@ -10,7 +10,7 @@ Systemd is used in order to have a command or program run when your device boots
 
 > ⚠️  If you wish to run the service with metrics enabled, you'll need to replace `--metrics:off` with `--metrics:on` in the service file below. See [here](./metrics-pretty-pictures.md) for more on metrics.
 
-Create a `systemd` service unit file -- `nimbus-eth2-pyrmont.service` -- and save it in `/etc/systemd/system/`.
+Create a `systemd` service unit file -- `nimbus-eth2-pyrmont.service` -- and save it in `/lib/systemd/system/`.
```
Comment (arnetheduck) [inline docs/the_nimbus_book/src/beacon-node-systemd.md:13]: https://unix.stackexchange.com/a/367237 - `/etc/` is supposedly the correct path, not sure why `/lib` enters the picture?

Comment (TennisBowling) [reply docs/the_nimbus_book/src/beacon-node-systemd.md:13]: here's the thing. `/etc/` is normally the correct path, yes. but if you want to be able to enable it on startup it makes a systemlink to `/etc/`, and so if you put it in /etc/, it won't be able to enable it on startup.
I had this problem myself and when I set it to lib I could enable it just fine.

**Pull request merged**